docs/v2/spec/2018_Public_LAR_Data_Dictionary.csv

```diff
@@ -0,0 +1,94 @@
+Ethnicity of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Integer,1 2 3 4,
+Race of Applicant or Borrower: 1,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 6 7,
+Race of Applicant or Borrower: 2,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Applicant or Borrower: 3,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Applicant or Borrower: 4,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Applicant or Borrower: 5,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Co-Applicant or Co-Borrower: 1,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5 6 7 8,
+Race of Co-Applicant or Co-Borrower: 2,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Co-Applicant or Co-Borrower: 3,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Co-Applicant or Co-Borrower: 4,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Co-Applicant or Co-Borrower: 5,Character Varying,1 2 21 22 23 24 25 26 27 3 4 41 42 43 44 5,
+Race of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Integer,1 2 3,
+Race of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Integer,1 2 3 4,
+Sex of Applicant or Borrower,Integer,1 2 3 4 6,
+Sex of Co-Applicant or Co-Borrower,Integer,1 2 3 4 5 6,
+Sex of Applicant or Borrower Collected on the Basis of Visual Observation or Surname,Integer,1 2 3,
+Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname,Integer,1 2 3 4,
+Age of Applicant or Borrower,Character Varying,<25 25-34 35-44 45-54 55-64 65-74 >74 8888,
+Age of Applicant >= 62,Character Varying,Yes No NA,
+Age of Co-Applicant or Co-Borrower,Character Varying,<25 25-34 35-44 45-54 55-64 65-74 >74 8888 9999,
+Age of Co-Applicant >= 62,Character Varying,Yes No NA,
+Income,Character Varying,NA,
+Type of Purchaser,Integer,0 1 2 3 4 5 6 71 72 8 9,
+Rate Spread,Character Varying,NA Exempt,
+HOEPA Status,Integer,1 2 3,
+Lien Status,Integer,1 2,
+Applicant or Borrower - Name and Version of Credit Scoring Model,Integer,1 2 3 4 5 6 7 8 9 1111,
+Co-Applicant or Co-Borrower - Name and Version of Credit Scoring Model,Integer,1 2 3 4 5 6 7 8 9 10 1111,
+Reason for Denial: 1,Integer,1 2 3 4 5 6 7 8 9 10 1111,
+Reason for Denial: 2,Integer,1 2 3 4 5 6 7 8 9,
+Reason for Denial: 3,Integer,1 2 3 4 5 6 7 8 9,
+Reason for Denial: 4,Integer,1 2 3 4 5 6 7 8 9,
+Total Loan Costs,Character Varying,NA Exempt,
+Total Points and Fees,Character Varying,NA Exempt,
+Origination Charges,Character Varying,NA Exempt,
+Discount Points,Character Varying,NA Exempt,
+Lender Credits,Character Varying,NA Exempt,
+Interest Rate,Character Varying,NA Exempt,
+Prepayment Penalty Term,Character Varying,NA Exempt,
+Debt-to-Income Ratio,Character Varying,<20% 20%-<30% 30%-<36% 50%-60% >60% Exact values between 36%-<50% NA Exempt,

```

Comment:
Can you move the exact values part into the notes column? 

Please do so for the MLAR dictionary as well.